webpages/js/metaanalyses.js

```diff
@@ -2981,12 +2981,14 @@
   // e.g. ['hidden0', 'col1', 'hidden2', 'hidden3', 'col4'].
   // Passing col1 will unhide hidden0, and passing col4 will unhide 2 and 3.
   function unhideColumns (e) {
-    var index = parseInt(_.findEl(_.findPrecedingEl(e.target, 'th'), '.fullcolinfo.popupbox').dataset.index);
+    var element = _.findEl(_.findPrecedingEl(e.target, 'th'), '.fullcolinfo.popupbox');
+    var index = 0;
 
-    // If we don't have a colIndex, it's the 'add column' button, so start at the end.
-    if (isNaN(index)) {
+    // If element == null or we don't have a colIndex, it's the 'add column' button, so start at the end.
+    if (element == null || ((index = parseInt(element.dataset.index)) && isNaN(index))) {
       index = currentMetaanalysis.columns.length-1;
-    } else {
+    }
```

Comment:
please put the else on the same line as `}`; we only use `else` on its own line when the blocks are bigger and we want better to separate them visually